Lab 1/Lab1.html

```diff
@@ -0,0 +1,221 @@
iUmYmZmZTUzCzMzMbGISZmZmZhOTMDMzM5uYhJmZmdnEJMzM7HVn7/CIXCq5VHKp5FLJpZJLJZdKLpVcKrlU9g6PMDttEmZm9rqzd3DMYlVZrCqLVWWxqixWlcWqslhVFqvKYlVZrCp7B8eYnTYJMzMzs4lJmJmZmU1MwszMXvfObayxs7nOzuY65zbWMDvtEmZm9rp3duMM+fw6+fw6ZzfOYHbaJczMzMwmJmFmZmY2MQkzMzOziUmYmZmZTUzCzMzMbGISZmZmZhOTMDMzM5uYhJmZmdnEJMzMzMwmJmFmZmY2MQkzMzOziUmYmZmZTUzCzMzMbGISZmZmZhOTOAWuXbvGtzo4OOCxxx7jC1/4AmZmZjYtiRPuU5/6FPfccw8v9/GPf5zf+q3fopTCYrHgAx/4AN/4xjcwMzOzaUicUMfHx9x999089NBDvNyzzz7LX/zFX/A3f/M33Hffffzd3/0dX/va13j00UcxMzOzaUicUA888AA333wz9957Ly+3trbGlStXWFtb45vW19e5fv06r0QSkpDE7u4uZmZmr0e7u7tIQhKSOKkSJ9TFixf56Ec/yk033cTL/eiP/ijvete7+KYvf/nLPP7447z//e/nlUQEEUFEsL29jZmZ2evR9vY2EUFEEBGcVIkTKqXE/8+///u/85u/+ZvcddddvPWtb8XMzMymIXFKffGLX+SXf/mX+Y3f+A3m8zlmZmY2HYlT6IknnuDDH/4wOWc+9KEPYWZmZtOSOGWee+45Lly4wCc+8Qne85738MILL/DCCy/w4osvYmZmZtOQOGUeeeQRvv71r/ORj3yEW2+9lVtvvZVbb72VP/iDP8DMzMymIXHCnT17lmvXrvFNv/d7v0dEEBFEBBFBRHDx4kXMzMxsGhJmZmZmE5MwMzMzm5iEmZmZ2cQkzMzMzCYmYWZmZjYxCTMzM7OJSZiZmZlNTMLMzMxsYhJmZmZmE5MwMzMzm5iEmZmZ2cQkzMzMzCYmYWZmZjYxCTMzM7OJSZiZmZlNTMLMzMxsYhJmZmZmE5MwMzMzm5iEmZmZ2cQkzMzMzCYmYWZmZjYxCTMzM7OJSZiZmZlNTMLMzMxsYhJmZmZmE5MwMzMzm5iEmZmZ2cQkzMzMzCYmYWZmZjYxCTMzM7OJSZiZmZlNTMLMzMxsYhKnwLVr1/hWzz33HI899hgRgZmZmU1L4oT71Kc+xT333MPLPfroo9xxxx2UUpjP51y6dAkzO1n2Do/IpZJLJZdKLpVcKrlUcqnkUsmlkksll8re4RFmNh2JE+r4+Ji7776bhx56iJd78cUX2dnZ4eGHH+a+++7jr//6r/n0pz/Nl770Jczs5Ng7OGaxqixWlcWqslhVFqvKYlVZrCqLVWWxqixWlcWqsndwjJlNR+KEeuCBB7j55pu59957ebl/+Id/YG1tjZ/4iZ/ghptvvpl3v/vdfO5zn+OVSEISktjd3cXMzOz1aHd3F0lIQhInVeKEunjxIh/96Ee56aabeLnj42N+8id/kpd7wxvewDiOvJKIICKICLa3tzEzM3s92t7eJiKICCKCkypxQqWU+HZefPFFUkq8XEqJl156CTM7uc5trLGzuc7O5jo7m+uc21jDzKYrccr84A/+IC+++CIv99JLL/H93//9mNnJdXbjDPn8Ovn8Ovn8Omc3zmBm05U4ZX7kR36Ef/qnf+Lljo6O+Jmf+RnMzMxsGhKnzM/93M9xw/7+Pjf8y7/8C0888QQ///M/j5mZmU1D4pRJKXH//fdzzz338MEPfpCtrS3+8A//kDe+8Y2YmZnZNCROuLNnz3Lt2jVe7p3vfCef//znefjhh3n66adpmgYzMzObjoSZmZnZxCTMzMzMJiZhZmZmNjEJMzMzs4lJmJmZmU1MwszMzGxiEmZmZmYTkzAzMzObmISZmZnZxCTMzMzMJiZhZmZmNjEJMzMzs4lJmNlk7R0ekUsll0oulVwquVRyqeRSyaWSSyWXSi6VvcMjzMxOgoSZTdbewTGLVWWxqixWlcWqslhVFqvKYlVZrCqLVWWxqixWlb2DY8zMToKEmZmZ2cQkzMzMzCYmYWYnxrmNNXY219nZXGdnc51zG2uYmZ1ECTM7Mc5unCGfXyefXyefX+fsxhnMzE6ihJmZmdnEJMzMzMwmJmFmZmY2MQkzMzOziUmYmZmZTUzCzMzMbGISZmZmZhOTMDMzM5uYhJmZmdnEJMzMzMwmJmFmZmY2MQkzMzOziUmYmZmZTUzCzMzMbGISp9SXvvQlHnvsMZ599lnMzMxsWhKn0Kc//Wl+7dd+jVIKv/M7v8Pv//7vY2ZmZtOROGVeeukl7r//fv70T/+U++67j7/8y7/kb//2b3n22WcxMzOzaUicQv/1X//FD/3QD3HDTTfdREqJ//zP/+SVSEISktjd3cXMzOz1aHd3F0lIQhInVeKUSSmxs7PDXXfdxaVLl7jzzju5/fbbecc73sEriQgigohge3sbMzOz16Pt7W0igoggIjipEqfQ008/zQ//8A/zpje9ibW1NQ4PD/mP//gPzMzMbBoSp8xnP/tZ/vEf/5FHHnmED3zgA1y5coUb/uRP/gQzMzObhsQpc3R0xGw24/u+7/v4pre85S0899xzmJmZ2TQkTpm3ve1tfP7zn+df//VfueGrX/0qTz/9NLfddhtmZmY2DYlT5q1vfSv33HMPv/qrv8oHP/hBfvEXf5Ff+IVf4Fd+5VcwMzOzaUicQrfffjtPP/00Dz/8ME8//TR33303ZmZmNh0JMzMzs4lJmJmZmU1MwszMzGxiEmZmZmYTkzAzMzObmISZmZnZxCTMzMzMJiZhZmZmNjEJMzMzs4lJmJmZmU1MwszMzGxiEmZm9qrtHR6RSyWXSi6VXCq5VHKp5FLJpZJLJZdKLpW9wyPM7LWTMDOzV23v4JjFqrJYVRarymJVWawqi1VlsaosVpXFqrJYVRaryt7BMWb22kmYmZmZTUzCzMzMbGIS
```

Comment:
This explanation for the negative sign is not clear to me.

The area under the curve of a standard normal distribution (mean 0, sigma 1)  gives the probability that your value lies between the two values that set the bounds of that area. The Z value is one of these bounds, the other depends on which table you're reading from (typically it'll be from the mean or from positive/negative infinity). For large Z, integer or otherwise, the area under the curve out towards infinity is very, very small. However, the norminv() function assumes that the area/probability you give it starts from negative infinity, and will find the Z to produce that area. If the area/probability you give it is less than 0.50, then the Z it gives you will be negative because Z will need to be to the left of the mean.

Reply:
so can I say since the sigma is the distance away from the mean, we can use (1- probability) to find sigma on the right side of the mean which will be the same as the left side of the curve.

Reply:
can i say the chance to find the negative sigma with given probability is the same as the chance to find the positive sigma with inverse probability.